pkcs11/src/backend/db/object.rs

```diff
@@ -480,6 +483,7 @@ pub fn from_cert_data(
         id: key_id.to_owned(),
         size: Some(length),
         mechanisms: vec![],
+        subject: Some(cert.tbs_certificate.subject.to_der().map_err(Error::Der)?),
```

Comment:
This is a bit of a duplicate to the filter attribute set in [line 451](https://github.com/Nitrokey/nethsm-pkcs11/blob/ea205049612ca9a8fd4fe20ed612cccc43d9438d/pkcs11/src/backend/db/object.rs#L456)

Reply:
OK